### PR TITLE
feature/47830 create web app service via Azure CLI

### PR DESCRIPTION
**Pull request by GuyHarwood**
Azure CLI shell script to create web apps and apply appropriate settings

### CODE REVIEW

File: deploy/web/create.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -e
+
+RESOURCE_GROUP=$1
+APP_NAME=$2
+APP_SVC_PLAN=$3
+IMAGE_NAME=$4
+IMAGE_VERSION=$5
+#DOCKER_USER=$6
```
Comment (jon-shipley) [inline deploy/web/create.sh:9]: Is there a reason to keep the commented out variables if they are not used?

**Review by jon-shipley — APPROVED** [review]

File: deploy/web/create.sh (hunk 1 above)
Comment (GuyHarwood) [reply deploy/web/create.sh:9]: no, i think now i have proven the 'lack of support' for it via the CLI, we will solely use the ACR.  will remove

**Review by jon-shipley — APPROVED** [review]
lgtm

**Comment by GuyHarwood** [comment]
QA Not required on this one.  will be tested as part of full environment creation script

**Pull request merged**